controllers/elfmachine_controller.go

```diff
@@ -626,11 +626,50 @@ func (r *ElfMachineReconciler) getVM(ctx *context.MachineContext) (*models.VM, e
 	return nil, errors.Wrapf(err, "failed to create VM for ElfMachine %s/%s", ctx.ElfMachine.Namespace, ctx.ElfMachine.Name)
 }
 
+// reconcileVMStatus attempt to recover the VM from an abnormal state.
```

Comment:
这个写得不对

Reply:
修改为 
```
// reconcileVMStatus make sures the VM in the Running status.
// 1. VM in STOPPED status will be powered on.
// 2. VM in SUSPENDED status will be powered off and power on.
// It will return true when VM status is not in STOPPED or SUSPENDED status.
```

Reply:
=> // reconcileVMStatus ensures the VM is in Running status.